app/controllers/api/v2/trends_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api
+  module V2
+    class TrendsController < V2::BaseController
+      include Foreman::Controller::Parameters::Trend
+
+      before_action :find_resource, :only => [:show, :destroy]
+
+      api :GET, "/trends/", N_("List of trends counters")
+      def index
+        @trends = resource_scope_for_index
+      end
+
+      api :GET, "/trends/:id/", N_("Show a trend")
+      param :trend_id, :identifier, :required => true
+      def show
+      end
+
+      api :POST, "/trends/", N_("Create a trend counter")
+      def create
+        params[:trend] ||= { }
+        @trend  = params[:trend][:trendable_type] == 'FactName' ? FactTrend.new(trend_params) : ForemanTrend.new(trend_params)
+        if @trend.save
+          process_success
+        else
+          process_resource_error
+        end
+      end
+
+      api :DELETE, "/trends/:id/", N_("Delete a trend counter")
+      param :id, :identifier, :required => true
+      def destroy
+        process_response @trend.destroy
+      end
+
+      def resource_scope_for_index(options = {})
+        resource_scope(options).paginate(paginate_options)
```

Comment:
in the resource_scope_for_index, there is usage in search_for method, 
and I can't use it because trends don't have this option

Reply:
> in the resource_scope_for_index, there is usage in search_for method,
and I can't use it because trends don't have this option

Please leave this comment in code.